lib/datasource/helm/common.ts

```diff
@@ -16,12 +16,13 @@ export function findSourceUrl(release: HelmRelease): RepoSource {
     return { sourceUrl: releaseMatch[1] };
   }
 
+  let homeMatch;
   if (release.home) {
-    const githubUrlMatch = githubUrl.exec(release.home);
-    if (githubUrlMatch?.groups && chartRepo.test(githubUrlMatch?.groups.repo)) {
+    homeMatch = githubUrl.exec(release.home);
+    if (homeMatch?.groups && chartRepo.test(homeMatch?.groups.repo)) {
```

Comment:
```suggestion
    homeMatch = githubUrl.exec(release.home)?.groups;
    if (homeMatch && chartRepo.test(homeMatch.repo)) {
```
 Make other code equivalent.